### ISSUE TITLE
Flake8 complains when having two blank lines between imports and classes with a comment inbetween.

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#587]
In GitLab by @normanjaeckel on Jun 1, 2017, 14:12

I use flake8 v3.3.0 (installed via pip).

This code fails

```python
import sys

# Some classes

class One:
    pass


class Two:
    pass


# Some other stuff

def foo():
    print(sys.path)
```

If I add a blank line between the import statement and the comment, flake9 runs correctly but isort fails. See https://github.com/timothycrosley/isort/issues/550.

The isort maintainer means that isort is correct according to PEP8.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jun 1, 2017, 16:48

This should be reported to [pycodestyle](https://github.com/pycqa/pycodestyle/issues). The error you're seeing is generated by that project.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jun 1, 2017, 16:48

closed

**Comment by asottile** [comment]
In GitLab by @normanjaeckel on Jun 2, 2017, 03:16

Thanks. See https://github.com/PyCQA/pycodestyle/issues/652
